core/utils/poly-look/src/visualisations/charts/bar-charts/horizontalBarChart/horizontalBarChart.js

```diff
@@ -315,6 +318,7 @@ export class HorizontalBarChart extends Chart {
     if (this._barValueColor) this._displayValues(barGroups, enteringBarGroups);
     else this.chart.selectAll(".bar-value").remove();
     if (this._groups) {
+      this.chart.selectAll(".group-label").remove();
```

Comment:
Here you're removing the group labels, is that the fix?

Reply:
this is also important because once the fix went through the group labels were being drawn twice